bookkeeper-server/src/test/java/org/apache/bookkeeper/test/BookieZKExpireTest.java

```diff
@@ -41,18 +43,92 @@
 /**
  * Test bookie expiration.
  */
+@Slf4j
 public class BookieZKExpireTest extends BookKeeperClusterTestCase {
 
     public BookieZKExpireTest() {
         super(0);
-        // 6000 is minimum due to default tick time
-        baseConf.setZkTimeout(6000);
-        baseClientConf.setZkTimeout(6000);
     }
 
+    /*
+    Should recover from request timeout.
+    */
+    @Test
     @SuppressWarnings("deprecation")
+    public void testBookieServerZKRequestTimeoutBehaviour() throws Exception {
+        // 6000 is minimum due to default tick time
+        System.setProperty("zookeeper.request.timeout", "6000");
```

Comment:
We have to do  System.clearProperty in a finally block

Reply:
fixed